requirements/collect_env_details.py

```diff
@@ -56,6 +57,7 @@ def info_packages():
         "pyTorch_version": torch.__version__,
         "pyTorch_debug": torch.version.debug,
         "pytorch-lightning": pytorch_lightning.__version__,
+        "lightning-app": lightning_app.__version__,
```

Comment:
Some users won't have it installed, right?

Reply:
This comes with lightning